README.rst

```diff
@@ -131,6 +131,11 @@ Core contributors
 Active contributors
 ...................
 
+* `Jake Rembish`_
+    * `UT Health San Antonio`_, USA
+
+.. _`Jake Rembish`: https://github.com/rembishj
+
```

Comment:
Bumped you up to the top of the active contributors list since you have been the most active in the repo apart from myself in the recent months.